config/settings.yml

```diff
@@ -76,6 +76,10 @@
   :concurrent_per_host: 1
   :scan_via_host: true
   :use_vim_broker: true
+:container_logging:
+  :by_provider_name:
+    :_default:
+      :kibana_route: logging-kibana
```

Comment:
I think having just one global setting is good enough to start with.

WRT this strategy to store per-provider settings I don't find it particularly interesting for several reasons:

1. it's hard to maintain
2. the association by name doesn't seems reliable enough
3. we're moving in a direction where we would like all these settings to be configurable with rest-api calls